BUILD.bazel

```diff
@@ -150,11 +150,13 @@ java_proto_library(
 # Deprecated: do not use
 proto_library(
     name = "reflection_proto_deprecated",
```

Comment:
This needs a `deprecation=....` param

Reply:
What would you like it to say? Should I add the same to the other as well?

Reply:
Yes, but I think since the java_proto_library rule is not a real rule.   If you can add it, please do.

Something like "Use reflection_proto instead".  I'm not so picky.

Reply:
Appears to work for both.